site/docs/4.3/components/popovers.md

```diff
@@ -250,6 +250,24 @@ Options can be passed via data attributes or JavaScript. For data attributes, ap
       <td>'scrollParent'</td>
       <td>Overflow constraint boundary of the popover. Accepts the values of <code>'viewport'</code>, <code>'window'</code>, <code>'scrollParent'</code>, or an HTMLElement reference (JavaScript only). For more information refer to Popper.js's <a href="https://popper.js.org/popper-documentation.html#modifiers..preventOverflow.boundariesElement">preventOverflow docs</a>.</td>
     </tr>
+    <tr>
+      <td>sanitize</td>
+      <td>boolean</td>
+      <td>true</td>
+      <td>Enable or disable the sanitization. If activated <code>'template'</code>, <code>'content'</code> and <code>'title'</code> options will be sanitized.</td>
```

Comment:
Oh for both tooltips and popovers, maybe add a note either here stating something to the effect of:

`Note: for security reasons this option cannot be supplied using a data attribute.`

Or if that becomes too verbose in the table, maybe adding something below the table:

```
Note: for security reasons the `sanitize`, `sanitizeFn` and `whiteList` options cannot be supplied using a data attribute.
```